.github/workflows/fuzz.yml

```
@@ -50,7 +50,7 @@ jobs:
           key: cache-${{ matrix.target }}-${{ hashFiles('**/Cargo.toml','**/Cargo.lock') }}
```

actionlint (1.7.7)

property "target" is not defined in object type {fuzz_target: string}

(expression)